Pulling updates from origin repository

chk0912 opened the pull request:
## Adding
- Avalance assets
- Other networks that we are not using right now
- Mainnet assets
- Fantom assets
- Polygon assets

## Updates
- BSC -> BNB
- Changes for mainnet tokens and logos

julianariel reviewed (commented):
nice, great work, README is very clear! 

I'm not entirely sure about the structure, I mean for now it's more than enough but looking at [TrustWallet assets repo](https://github.com/trustwallet/assets) it's interesting that they divide it in **blockchains** and **dApps**. Of course that is a much more advanced repo with indexed data and lots of work but thinking in the long term, to prevent changing URL's again in the future we could think of a similar structure at least for main folders.

What do you think?

julianariel commented:
Mmm it's a good point, we should probably allow both formats or like you said, expect network logos on SVG and token logos in PNG, that could work.

Regarding other repos, TrustWallet is the most complete one that I've found, we are actually using it on background today. Doing some research I found that [Uniswap](https://github.com/Uniswap/assets) , [Compound](https://github.com/compound-finance/tokens),  [1Inch](https://github.com/1inch/assets) and [SushiSwap](https://github.com/sushiswap/assets) forked the same repo. [Pancake](https://github.com/pancakeswap/pancake-toolkit/tree/master/packages/token-lists) seem to have a similar JSON list with links pointing to their own domain, on PNG format too. 

Now that I see this I'm not entirely sure if we should stick with the SVG format 😖

JulesGuesnon commented:
So I dug a bit into the repos and I think that the svg will be a bad idea. It would probably end up with someone who would have to turn png into svg.

Imo, let's stick with trustwallet architecture and png images, it seems like the most scalable choice

julianariel commented:
Yes I agree, let's do that then, you can update the structure, images and README to start using it with PNG 👌

kshamenkgil commented:
> Oh, thanks for the link, didn't know about that repo. Actually it looks like a good structure and it can be a good idea to split by blockchain.
> 
> Also while I browsed their assets, I noticed that the SVG only policy is not adapted at all. Well, maybe it can work for network tokens, but not for tokens like [this one](https://github.com/trustwallet/assets/blob/master/blockchains/avalanchec/assets/0xd039C9079ca7F2a87D632A9C0d7cEa0137bAcFB5/logo.png)
> 
> I think it's a good inspiration, but do you know other assets repo?

Hey guys, just saw this and wanted to add something regarding what @julianariel said, I strongly think that the approach here is as Juli mentioned to follow a similar structure as TrustWallet does, as currently all of our Token metadata structured is based on this. So, what I think it would be good to do right now is to actually fork this from the TrustWallet repo, replacing the images that we need for these type of assets. Why I say this, is because otherwise, we would end up having in the extension two different repositories for exactly the same thing and a new dependency for only a couple of currencies.

unanoc committed:
Fixes (sanity and consistency, auto)

SolutionPr committed:
Add UZYTH (BEP20) (#18388)

* Info for token UZYTH

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Azacora committed:
Add AFFINITY (BEP20) (#18374)

* Info for token AFFINITY

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

chuobacca committed:
Info for token Alpine F1 Team Fan Token (#18216)

yenthanh132 committed:
Info for token ASPO World (#18395)

Commit:
Add AutoCrypto (BEP20) (#18311)

* AutoCrypto info.json

* AutoCrypto logo.png

* Delete logo.png

Deleting to update the image

* AutoCrypto logo

* Info for token AutoCrypto

zachzwei committed:
Update info.json

update BIFI description

RealShibaDogeDev committed:
Add ShibaDoge (ERC20) (#18302)

* Info for token ShibaDoge

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

kodicoinofficial committed:
Add KODI COIN (BEP20) (#18427)

* Add KODI logo and info

Kodi provides its users with the first ever crypto based Entertainment Network. Parallel to this, Kodi is operating a one stop shop Advertisement/Marketing Agency known as ‘PITCH’, which will function as a subsidiary company. Both entities will encompass a use case that will directly benefit $KODI holders. This whitepaper provides a detailed description of the use case, tokenomics, team, the future plans of PITCH and its utility token ($KODI).

* Add files via upload

Kodi provides its users with the first ever crypto based Entertainment Network. Parallel to this, Kodi is operating a one stop shop Advertisement/Marketing Agency known as ‘PITCH’, which will function as a subsidiary company. Both entities will encompass a use case that will directly benefit $KODI holders. This whitepaper provides a detailed description of the use case, tokenomics, team, the future plans of PITCH and its utility token ($KODI).

* Add KODI logo to Trust Wallet

Kodi provides its users with the first ever crypto based Entertainment Network. Parallel to this, Kodi is operating a one stop shop Advertisement/Marketing Agency known as ‘PITCH’, which will function as a subsidiary company. Both entities will encompass a use case that will directly benefit $KODI holders. This whitepaper provides a detailed description of the use case, tokenomics, team, the future plans of PITCH and its utility token ($KODI).

* Add KODI logo to Trust Wallets

Kodi provides its users with the first ever crypto based Entertainment Network. Parallel to this, Kodi is operating a one stop shop Advertisement/Marketing Agency known as ‘PITCH’, which will function as a subsidiary company. Both entities will encompass a use case that will directly benefit $KODI holders. This whitepaper provides a detailed description of the use case, tokenomics, team, the future plans of PITCH and its utility token ($KODI).

* Info for token KODI COIN

* Delete info.json

* Delete logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

cryptomanz1 committed:
Migration for Luckypig (BEP20) (#18447)

* Delete logo.png

* Update info.json

* Create logo.png

* Create info.json

* Update info.json

* Update logo.png

* Fixes (sanity and consistency, auto)

Co-authored-by: cryptomanz <cryptomanz@users.noreply.github.com>

dhabita committed:
update audit NBG (#18428)

* up

* up

* up

* Update info.json

* up

* Update info.json

* Update info.json

* Update logo.png

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

axengine committed:
Add Singularity SGLY Coin (BEP20) (#18386)

Commit:
Fixes (sanity and consistency, auto)

hewigovens committed:
Update bnb chain logo (#18457)

hewigovens committed:
Update smart chain logo (#18481)

stakepile committed:
add stakepile logo (#18484)

Commit:
Fixes (sanity and consistency, auto)

finodao committed:
Updated the better logo (#18492)

* Add FINO DAO token and description

Add FINO DAO token and description and logo

* Update info.json

* Delete logo and push a new one

* Update a new logo

* delete logo

* Add new logo

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

web3doge committed:
Info for token WEB3 Doge (#18498)

calvingreen committed:
update info (#18509)

* add MiloInu logo and info

add MiloInu logo and info

* add MiloInu logo and info

add MiloInu logo and info

* add MiloInu logo and info

add MiloInu logo and info；18446

* update token info

add coingecko and coinmarketcap

* update info; PR is #18446

add coinmarketcap and coingecko;
we have 10,251 holders and 82,721 transfers;
https://github.com/trustwallet/assets/pull/18446

* Add files via upload

* Add files via upload

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

SeptiyanAndika committed:
Add NBT (BEP20)  (#18503)

* Add NBT (BEP20)

* add NBT tags

* update tags informations and logo size

cryptomanz1 committed:
Mirgation for Santa Floki v2 (BEP20) (#18511)

* Delete logo.png

* Update info.json

* Create info.json

* Create logo.png

* Fixes (sanity and consistency, auto)

Co-authored-by: cryptomanz <cryptomanz@users.noreply.github.com>

fqswap committed:
Add FqSwap (BEP20) (#18527)

* Create info.json

* Add files via upload

* Update info.json

add tags

* Info for token FqSwap

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

MetaDogeContract committed:
Info for token MetaDoge V2 (#18501)

cryptomanz1 committed:
Migration for Floki Inu (BEP20) (#18622)

* Create logo.png

* Create info.json

* Update info.json

* Update logo.png

* Update info.json

* Delete logo.png

* Update tokenlist.json

* Fixes (sanity and consistency, auto)

Co-authored-by: cryptomanz <cryptomanz@users.noreply.github.com>

Commit:
Add Defina Token (BEP20) (#18393)

* Add logo and info for DEFINA Token (BEP20)

Logo and Description uploaded for DEFINA Token (FINA).

* Info for token Defina Token

* Delete info.json

* Delete logo.png

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

darkforestcap committed:
Info for token MetaPortal Gaming Index (#18533)

KibaInuu committed:
Info for token Quantic (#18636)

Devalicious committed:
Add SHIP (BEP20) (#18639)

* Add SHIP (BEP20)

* Corrected errors

Removed NFT tag, updated discord link

* Update info.json

* Removed Instagram, Youtube

* Update info.json

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

jetrich committed:
Info for token DYOR Token (#18554)

kinmengchan committed:
Info for token FROYO (#18611)

Cryptocool1 committed:
add links to memes token (#18654)

* add links

* Update info.json

* Fixes (sanity and consistency, auto)

Co-authored-by: Cryptocool1 <Cryptocool1@users.noreply.github.com>

cryptomanz1 committed:
Migration for BNBPay (BEP20) (#18655)

* Update info.json

* Delete logo.png

* Fixes (sanity and consistency, auto)

Co-authored-by: cryptomanz <cryptomanz@users.noreply.github.com>

bnbpay-dev committed:
Add PayBolt Token (ERC20, BEP20, POLYGON) (#18618)

* Add PayBolt Token (ERC20, BEP20, POLYGON)

* Add PayBolt Token (ERC20, BEP20, POLYGON) - Corrected youtube URL

* Add PayBolt Token (ERC20, BEP20, POLYGON) - Corrected facebook URL

Co-authored-by: mHat <>

LunarLegionBsc committed:
Info for token Lunar (#18651)

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Commit:
Add BLP & GMPD (BEP-20) (#18649)

* Add BLP & GMPD logo and info

Add BullPerks and GamesPad tokens info to TrustWallet

* Add BLP & GMPD logo and info

Add BullPerks and GamesPad tokens info to TrustWallet (With links)

* Add BLP & GMPD logo and info

Add BullPerks and GamesPad tokens info to TrustWallet (Fixed JSON)

* Add BLP & GMPD logo and info

Add BullPerks and GamesPad tokens info to TrustWallet (Removed tags)

* Add BLP & GMPD logo and info

Add BullPerks and GamesPad tokens info to TrustWallet (Added tags)

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

gotenksnull committed:
Add RUBY (BEP20) (#18658)

* Info for token RUBY

* Update info.json

removed empty discord link

ninnekodev committed:
Add NINO logo and info (#18681)

* Add NINO logo and info

Add logo and info of Ninneko token

* Add tags

Add tags to info

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

QuantumAssets committed:
Add Quantum Assets Token (BEP20) (#18662)

* Info for token Quantum Assets Token

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

TokenTravelCare committed:
Add TRAVEL logo and info (#18668)

* Add TRAVEL logo and info

Hello TrustWallet team,
Please add TravelCare logo in Trust Wallet.
Thank you.

* Update info.json

* Update info.json

Links and tags added

* Update info.json

Links and Tags added

* Update info.json

Tags and Links added. Thanks.

* Update info.json

Added Tags and links, thanks

* Update info.json

Added tags and links, thanks!

* Update info.json

Links and tags added.

* Update info.json

Links and tags added correctly. thanks.

* Update info.json

Links and tags added correctly. thanks.

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

cryptomanz1 committed:
Migration  for Bigh Bull (BEP20) (#18735)

* Update info.json

* Delete logo.png

* Fixes (sanity and consistency, auto)

Co-authored-by: cryptomanz <cryptomanz@users.noreply.github.com>

Commit:
Info for token BIGH BULL (#18698)

KibaInuu committed:
Info for token Baby Catcoin (#18736)

Enhancedtoken committed:
Info for token ENHANCE (#18737)

CliffordinuToken committed:
Add Clifford Inu (CLIFF) (ERC20) (#18739)

* Info for token Clifford Inu (CLIFF)

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

ThanhIres committed:
Add BEM (BEP20) (#18773)

* Update BEM Token logo

* Info for token BEM

* Delete info.json

* Delete logo.png

* Update logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

sandhya1349 committed:
Info for token LuckyBlock (#18780)

zombieworldz committed:
what next step to complete the task (#18591)

* Add ZwZ logo and info

submit info to trust wallet

* update info.json

* Update info

* update info

* update info

* Revert "update info.json"

This reverts commit f05548c2a4a224430810a7905b7c273db86c3057.

Co-authored-by: bruce <ioloi.trans@gmail.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

Cattokenn committed:
Add CatBoy (BEP20) (#18788)

* Info for token CatBoy

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

ChainEngineer1 committed:
Update CHN to XCN (#18789)

* add info

* add logo

* update CHN logo

* update token explorer

* update description

* update token info

* update CHN token

* update token address

* update XCN token

* add tag

* update XCN address

* update tag

PankajTMB committed:
Add BITLIVE (BEP20) (#18389)

* Info for token BITLIVE

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Socaverse committed:
Add SOCA logo and info (#18818)

* Add SOCA logo and info

Add SOCA logo and info

* Update info.json

* Add SOCA logo and info(BEP20)

Add SOCA logo and info

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

meowttt7 committed:
Add CBT logo and info (#18823)

* Add CBT logo and info

Request to add CyBloc Battery Token

* Update info.json

* Update info.json

Update discord link

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

manti committed:
oddz logo updated (#18781)

* oddz logo updated

* removed youtube link

Commit:
Fixes (sanity and consistency, auto)

GoBlockchain2022 committed:
Add: Go Crypto Me (BEP20) (#18798)

* Adding assets

* Update info.json

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

ramsty committed:
Fixes (sanity and consistency, auto)

bronejoost committed:
Adding 4JNET logo to Trust Wallet (#18819)

* Add files via upload

* Add logo in Trust Wallet

Not sure this is possible, also 0xc942CEee2D86E3e066Ac1f97044E911ced2fe314 (NFT address) needs the logo

* Delete info.json

* Delete logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

cryptomanz1 committed:
Add links for ODDZ (BEP20), (ERC20) (#18896)

* Update info.json

* Update info.json

* Fixes (sanity and consistency, auto)

Co-authored-by: cryptomanz <cryptomanz@users.noreply.github.com>

Rainhard-1 committed:
Update Website & CMC (#18901)

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

CMNOfficial committed:
Add Fairface Token (BEP20) (#18899)

* Info for token Fairface Token

* Update logo.png

* Update info.json

Co-authored-by: ramsty <91697270+ramsty@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

VoltInu committed:
Add Volt Inu (BEP20) (#18917)

* Update VOLT logo and details

Update VOLT logo and details

* Add VOLT INU logo and info

Add VOLT INU logo and info

* Info for token Volt Inu

* Update logo.png

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

Cattokenn committed:
Info for token Elvantis (#18926)

gusiri committed:
Update Splyt (SHOPX) Logo (ERC20)  (#18929)

* Info for token Splyt (SHOPX)

* Update info.json

* Update info.json

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Cryptocool1 committed:
Migration for Mind Music (#18937)

* Update info.json

* Delete logo.png

Cattokenn committed:
Info for token Mind Music (#18936)

unanoc committed:
Add OASIS token type (#18947)

vorotech committed:
Update s3_upload.yml (#18962)

assume deployment role instead of aws creds

issaelzein committed:
adding KATA (Katana Inu) (#18972)

* Add files via upload

* Add files via upload

* Add files via upload

* Add files via upload

* Add files via upload

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

HarePlus committed:
Add Hare Plus (BEP20) (#18986)

* Add HARE PLUS Logo

Add HARE PLUS Logo

* Add Hare Plus Logo

Add Hare Plus Logo

* Add HarePlus Logo

Add HarePlus Logo

* Add Hare Plus Logo

Add Hare Plus Logo

* Update Hare Plus Logo

Update Hare Plus Logo

* Info for token Hare Plus

* Delete info.json

* Delete logo.png

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

bjt54 committed:
Info for token ApeCoin (APE) (#19022)

AmazingDogebsc committed:
Add AmazingDoge logo and info (#19030)

* Add AmazingDoge logo and info

Please help to add AmazingDoge's logo and info

* Add AmazingDoge logo and info

Submit AmazingDoge's logo and info

* Update info.json

* Add AmazingDoge logo and info

Add AmazingDoge logo and info

* Add AmazingDoge logo and info

Add AmazingDoge logo and info

* Add AmazingDoge logo and info

Add AmazingDoge logo and info

* Update info.json

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

ChainEngineer1 committed:
Update XCN description (#19015)

* add info

* add logo

* update CHN logo

* update token explorer

* update description

* update token info

* update CHN token

* update token address

* update XCN token

* add tag

* update XCN address

* update tag

* update token symbol

zachzwei committed:
add_ERC20_APE (#19040)

* add_ERC20_APE

added APE ERC20 to tokenlist

* Update tokenlist.json

defininja committed:
Add GAMMA (BEP20) (#16148)

* Add Planet Finance's GAMMA to Trustwallet.

GAMMA is listed on CoinGecko - https://www.coingecko.com/en/coins/green-planet
Our protocol already has ~ $650 million total value locked in it
https://blue.planetfinance.io/
https://green.planetfinance.io/
https://defillama.com/protocol/planet-finance
Our other asset AQUA is already listed in Trustwallet assets (https://github.com/trustwallet/assets/tree/master/blockchains/smartchain/assets/0x72B7D61E8fC8cF971960DD9cfA59B8C829D91991)
I have also transferred 1000 TWT - https://explorer.binance.org/tx/AE9E26152451984C3BA0EBBF842179DC5A12DAD2E34E5EFDF61159CEF378E992
Few credible articles on BSC.new
https://www.bsc.news/post/green-planet-the-first-decentralized-money-market-with-discount-levels
https://www.bsc.news/post/planet-finance-bringing-defi-to-the-next-billion-users
Can you please add support for GAMMA, many people in our community use Trust Wallet and it's important that GAMMA shows correctly?

* Info for token GAMMA

* Delete logo.png

* Delete info.json

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

dark-sizter committed:
Add VEKTOR (BEP20) (#19046)

* Info for token VEKTOR

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

PearDAOCommunity committed:
Add PEX logo and info (#19023)

* Add PEX logo and info

Adding info for BEP20 token 'PEAR DAO (PEX)'.

* Update info.json

Adding links and tags

* Update info.json

Adjust tags

* Update info.json

Adjust tags

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

minerbloxoffi committed:
MinerBlox token (MBLOX) re-apply (original request PR #18519) (#19045)

* Minerblox token (MBLOX) logo.png and info.json

Minerblox token (MBLOX) logo.png and info.json for add our BEP20 token to trustwallet

* Update info.json with links and tag

* Correct ] on links

* Correct coingecko link and tag

* correct coingecko link

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

bodatoken committed:
Update Boda Token Logo (#19013)

* Add files via upload

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Delete logo.png

changing to new logo

* Add files via upload

* add tag

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

geekbtc committed:
Add SpacePi Token (BEP20) (#19021)

* Info for token SpacePi Token

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

ginoizm committed:
add Kunci Coin (#18390)

* 0x6cf271270662be1C4fc1b7BB7D7D7Fc60Cc19125/

* add tags. fix facebook URL.

* fix tags

KINGFOREVERBSC committed:
Add KING FOREVER (BEP20) (#18377)

* Info for token KING FOREVER

* Update info.json

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: ramsty <91697270+ramsty@users.noreply.github.com>

AtlantisFi committed:
Add Atlantis (BEP20) (#19070)

* Update Token logo to Trust Wallet

Link to 
https://coinmarketcap.com/currencies/atlantis-finance/

* Info for token Atlantis

Commit:
Add TOSKA (BEP20) (#19044)

* Info for token TOSKA

* Update logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

zachzwei committed:
add_AIRT_BEP20 (#19074)

added AIRT to tokenlist

AsserFawzi committed:
Add AXL INU (AXL) (BEP20) (#19103)

* Add AXL INU Logo and Info

Add AXL INU Logo and Info

* Add AXL INU BEP20 Logo and Info

Add AXL INU BEP20 Logo and Info

* Update info.json

* Update info.json

* Checksum

* Checksum

* Info for token AXL INU (AXL)

* Delete info.json

* Delete logo.png

* Update info.json

* Update logo.png

* Update logo.png

* Update info.json

* Create info.json

* Create logo.png

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

* Delete info.json

* Delete logo.png

* Rename blockchains/ethereum/assets/0x25b24B3c47918b7962B3e49C4F468367F73CC0E0 /info.json to blockchains/ethereum/assets/0x25b24B3c47918b7962B3e49C4F468367F73CC0E0/info.json

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

aiyou2 committed:
Add MetaHouse logo and info (#19076)

* Add MetaHouse logo and info

Add MetaHouse logo and info

* Update info.json

* add metahouse info

add metahouse info

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

Bitgert committed:
Info for token Centcex (#19117)

Gigablocks committed:
Add GIGA BLOCK (BEP20) (#19128)

* Info for token GIGA BLOCK

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Cattokenn committed:
Info for token Green Chart (#19167)

deepspacebsc committed:
Update DEEPSPACE (BEP20) Listing and Add dapp Logo and Request dapp Listing (#19178)

* update DEEPSPACE (DPS) Listing info

added tag and updated listing info

* Update info.json

* Add files via upload

* Update info.json

fixed social links

* Update info.json

* Update info.json

* Update info.json

zachzwei committed:
add_ERC20_STG (#19216)

ArtemGoryunov committed:
Green Metaverse Token (GMT) SPL (#19224)

megavalidator committed:
kava: validators: add Mega (#19221)

https://www.mintscan.io/kava/validators/kavavaloper1st7akgudd4d3x5t4zd3kn0fg948zn9yf4qpa0m

Co-authored-by: nodesbygirls-validator <hello@nodesbygirls.com>

tinyworldgame committed:
Add Tiny Coin (BEP20) (#19197)

* Info for token Tiny Coin

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

mysterboxio committed:
Add Mystery (BEP20) (#19255)

* Please add MYST logo and info to Trust Wallet

Requesting to add the "Mystery" $MYST logo and token info to trust wallet, thank you so much! 

-Nelson
CEO of Mystery

* Please add MYST info and logo to Trust Wallet

Please add "Mystery" (MYST) information and logo to Trust Wallet. 

Thank you so much!

Chris Nelson
Mystery CEO
team@mysterbox.io
(360)953-2593

* Info for token Mystery

* Delete info.json

* Delete logo.png

* Delete info.json.txt

* Delete logo.png.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

metaniagames committed:
metaniagames assets added (#19213)

* metaniagames assets

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

DEMONNNX committed:
Add DEMON (BEP20) (#19203)

* Info for token DEMON

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Qatar2022token committed:
Add Logo Fwc (#19241)

Holders:
9,375 addresses
Transfers:
86,772

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

nntam2013 committed:
Update OPEN token asset (#19262)

* update token info

* add tags

* update tags

sm-dasset committed:
Add Rune Shards (RXS) (BEP20) (#19260)

* Rune info and logo

Add Rune Game asset

* Runee links added

added links

* remove discord

* Update to RXS

* Updated

Fixed discord URL, removed tag

* Add Rune Shards (RXS) (BEP20)

* Delete old rune directory

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

SymmaBlockchain committed:
Add ELGOS Token (BEP20) (#19282)

* Info for token ELGOS Token

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

unanoc committed:
Add oasis explorer (#19295)

youwhoyou committed:
Info for token YOUWHO (#19297)

MetaxizDev committed:
Submit MEXI logo token (#19299)

* Submit MEXI logo token

- Project has a website and a detailed white paper. A clear roadmap, tokenomics and use case is a must. => done
- Has a social media presence and support team. No fake followers or bots. => done
- A full token audit is completed by reputable security audit groups. Partial audits are subject for review. => done
- No plagiarized content from other projects or companies. Copying names or logos are strictly prohibited. => done
- Token should be listed on price tracking sites like CoinmarketCap => done
- Detailed Token Information => done 
- Minimum 10,000 token holders and 15,000 transactions (Airdrop tokens excluded). This requirement is subject for review based on the project. => done

* Add the links and tag

* Update youtube link

* Update youtube link

* Update coingecko link

* Update coingecko link

* correct coingecko link

* remove invalid link

* Remove pair link

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

Meta-UFO committed:
Add MetaUFO LOGO (#19302)

* Add MetaUFO Logo

Add MetaUFO Logo

* Update info.json

* Update info.json

* Update info.json

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

artradeapp committed:
Add ATR logo and info (#19315)

* Add ATR logo and info

Request for adding ATR token to Trust Wallet assets in smartchain

* update file with links and tags

 Add  links and tags into info.json

* update file info.json

Delete Reddit link

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

iicc1 committed:
Add Stakely Cosmoshub, Osmosis, Polkadot & Terra validators (#19326)

* feat: add Stakely cosmoshub validator

* feat: add Stakely Osmosis validator

* feat: add Stakely Polkadot validator

* feat: add Stakely Terra validator

DTal621 committed:
New and updated Branded Dollars from ICHI (#19269)

* New and updated Branded Dollars from ICHI

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update info.json

* Update logo.png

* Update logo.png

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

smart2022 committed:
Add Frz solar system (BEP20) (#19244)

* Frz solar system (Frzss) BEP20 token logo & info 

{
 "name": "Frz solar system",
    "website": "https://frzss.com/",
    "description": "the main purpose of the FRZS token is to strengthen the solar energy industry and expanding this industry to help electricity supply for not a far future also with cooperation with companies, we have focused on reduce electricity generation costs",
    "explorer": "https://bscscan.com/token/0xac41fb8013c0b63588fc63997785a5d79e73eb28",
    "type": "BEP20",
    "symbol": "Frzss",
    "decimals": 18,
    "status": "active",
    "id": "0xaC41Fb8013c0B63588fC63997785A5d79E73eb28"
 Coin Market Cap: https://coinmarketcap.com/currencies/frz-solar-system/
 Coin Gecko : https://www.coingecko.com/en/coins/frz-solar-system
 Website : https://frzss.com/
 White paper : https://frzss.com/White-Paper/
 Telegram : http://t.me/frzsstoken
 Twitter : https://twitter.com/frztoken?t=50fc06_SxNW9rpEaR8-1cQ&s=09
}

* Frz solar system (Frzss) BEP20 token logo & info

"name": "Frz solar system",
    "website": "https://frzss.com/",
    "description": "the main purpose of the FRZS token is to strengthen the solar energy industry and expanding this industry to help electricity supply for not a far future also with cooperation with companies, we have focused on reduce electricity generation costs",
    "explorer": "https://bscscan.com/token/0xac41fb8013c0b63588fc63997785a5d79e73eb28",
    "type": "BEP20",
    "symbol": "Frzss",
    "decimals": 18,
    "status": "active",
    "id": "0xaC41Fb8013c0B63588fC63997785A5d79E73eb28"
 Coin Market Cap: https://coinmarketcap.com/currencies/frz-solar-system/
 Coin Gecko : https://www.coingecko.com/en/coins/frz-solar-system
 Website : https://frzss.com/
 White paper : https://frzss.com/White-Paper/
 Telegram : http://t.me/frzsstoken
 Twitter : https://twitter.com/frztoken?t=50fc06_SxNW9rpEaR8-1cQ&s=09
Number of holders : 11,489 addresses
#18774

* Info for token Frz solar system

* Delete info.json

* Delete logo.png

* Update info.json

* Delete info.json

* Delete logo.png

* Update info.json

* Create Frzss.sol

* Add files via upload

* Delete Frzss.sol

* Delete info.json

* Delete logo.png

* Update logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

optout21 committed:
Add KuCoin Community Chain logo (#19362)

jrhender committed:
add energyweb to blockchains (#19334)

* add energyweb to blockchains

* fix(energyweb): remove www from reddit url

* fix(energyweb): docs instead of documentation

K-Birdman committed:
Update CAN BEP20 json.info token details (#19353)

* Update BEP20 json.info token details

The BEP20 token name was incorrect and the description field was empty.

* Update website to CanWork.io

Second edit for website (CanWork.io) json.info

CYLUMFINANCE committed:
Add Cylum Finance (BEP20) (#19361)

* Info for token Cylum Finance

* Update info.json

* Update info.json

* Update info.json

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

kevinyliu committed:
add Wormhole assets (#18652)

* add Wormhole assets

* adjust image size, remove links

* fix LUNA id

* change id to checksum format

* Delete logo.png

* Delete info.json

* Create logo.png

* Create info.json

* Delete logo.png

* Delete info.json

* Create logo.png

* Create info.json

* Create logo.png

* Create info.json

* Delete logo.png

* Delete info.json

* Create logo.png

* Create info.json

* Delete logo.png

* Delete info.json

* Create info.json

* Create logo.png

* Delete info.json

* Delete logo.png

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

* Create info.json

* Create logo.png

* Delete info.json

* Delete logo.png

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

* Update info.json

* Update info.json

* Delete logo.png

* Delete info.json

* Create logo.png

* Create info.json

* Create logo.png

* Create info.json

* Delete logo.png

* Delete info.json

* Create info.json

* Create info.json

* Create logo.png

* Delete info.json

* Delete logo.png

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

* Create info.json

* Create logo.png

* Delete logo.png

* Delete info.json

* Delete info.json

* Delete logo.png

* Create info.json

* Create logo.png

* Delete logo.png

* Delete info.json

* Create logo.png

* Create info.json

* Delete info.json

* Create logo.png

* Create info.json

* Delete logo.png

* Delete logo.png

* Delete info.json

* Create logo.png

* Create info.json

* Delete info.json

* Create logo.png

* Create info.json

* Delete logo.png

* wormhole -> portal

Co-authored-by: Kevin Liu <kevinliu@Kevins-MacBook-Pro.local>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

bjt54 committed:
Fixes (sanity and consistency, auto)

adadao-org committed:
Add Adadao info and logo (#19381)

* Add Adadao info and logo

Add Adadao BEP-20 logo and info

* Update info.json

* Update info.json

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

MrLightspeed committed:
Info for token Puli (#19395)

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

METAWHALEWORLD committed:
Add MTW Logo and Add Info (#19388)

* Add Logo and Information

Please Add Logo and information

* Add Logo and Information

Add Logo and Information

* Add Logo and Information

Please Add Logo and Information

* Add Logo and Update Info

Please add MTW Logo and Update Info

Fee Paid References PR No - #19007

* Update info.json

* Add Logo and Update Info

Add MTW Logo and Update Info

Fee already paid PR No #19007

* Add  MTW Logo and Update Info

 Add MTW Logo and Update Info

MTW Is BEP20 Token

Fee Already Paid PR No #19007

* Add MTW Logo and Update Info

Add MTW Logo and Update Info

Fee Already Paid PR No #19007

* Add MTW Logo and Update Info

Add MTW Logo and Update Info

Fee Already Paid PR No #19007

All Changes are Done

MTW is BEP20 Token

Holder and Transaction are checked for References https://bscscan.com/token/0xD3Ac199E6e6A1668ED6566b6F6dcdf7641868731

* Update info.json

* Add MTW Logo and Update Info

Add MTW Logo and Update Info

Fee already Paid PR No #19007

Token Holders and Transactions are Checked Link :-https://bscscan.com/token/0xD3Ac199E6e6A1668ED6566b6F6dcdf7641868731

* Info for token Hare Plus

* Delete logo.png

* Add MTW Logo and Info

Add MTW Logo and Update Info

Fee Already Paid PR No #19007

All Changes are Done

MTW is BEP20 Token

PR #19053 

Required Smart Contract Audit Report Here is the link https://drive.google.com/file/d/1jTF5slWqhkMkE4yXoUm-ofLN0qvGpQdc/view?usp=sharing

Please Do as early as Possible and add logo on trust wallet

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: ramsty <91697270+ramsty@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

cryptomanz1 committed:
Update logo.png (#19421)

mikecryptobull committed:
Update STARS (BEP 20) token Logo and Info (#19393)

* Update STARS (BEP 20) token Logo and Info

Uploading Logo to be added to Trust Wallet in logo.png also an update in information contained within info.json

* Update info.json.json

* Rename info.json.json to info.json

* Create logo.png

* Delete logo.png.png

* Update info.json

* Update info.json

* Update info.json

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

bankteller-eth committed:
Add Elephant Money Stable (BEP20) (#19420)

* Info for token Elephant Money Stable

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

timvanp committed:
Pleasure Coin (NSFW) added (#19444)

* Pleasure Coin (NSFW) added

* Fixed errors (NSFW)

* Fixed tags (NSFW)

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

CryptoBotnik committed:
Add WEB3 (BEP20) (#19447)

* Add WEB3 (BEP20)

Adding info for BEP20 token 'WEB3'.

Please see https://github.com/trustwallet/assets/pull/19187 for payment details.

In response to the questions:

- The purpose of the token is to provide a high, stable and fixed yield to our users whilst bridging the gap between web2 and web3 for the masses.
- The token was distributed through an IDO, we also airdropped private sale buyers. It is currently available on PancakeSwap with centralised exchanges to come.
- Yes, the second largest ever presale on PinkSale raising over 6000 BNB: https://www.pinksale.finance/#/launchpad/0x8ABC3Be97Eb5C0ea271DB82694483Fd20b9ac281?chain=BSC
- It is currently available on PancakeSwap with centralised exchanges to come: https://pancakeswap.finance/info/token/0x333fd139caef6aa31056cc905987b77b1044d259
- Yes multiple audits with CertiK in progress:
        - https://github.com/AuditRateTech/Smart-Contract-Audits/blob/main/The_Web3_Project_0x333FD139cAeF6Aa31056cC905987b77B1044d259.pdf
        - https://www.certik.com/projects/the-web3-project
        - https://github.com/coinscope-co/audits/blob/main/2-web3/audit.pdf

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

Qatar2022token committed:
Change logo by community (#19457)

* Add Logo Fwc

Holders:
9,375 addresses
Transfers:
86,772

* Update logo by community (change logo)

Change logo

* Change logo by Community

Change logo

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

MONETASGLOBAL committed:
Change Contract Address (#19452)

* Change Contract Address

For Ref PR No #19382

Regarding Contract Address Change

Please Refer Issue No :-#19363 , #19374

Old Contract Address :- 0xE440b9af0AE99174b3FAaB87174E31634D135f6d

New Contract Address :- 0x10adF50e15611d5A4DE3bd21F0DB7f3491A8ae0f

* Change Contract Address

Reference PR #19387

For Ref PR No #19382

Regarding Contract Address Change

Please Refer Issue No :-#19363 , #19374

Old Contract Address :- 0xE440b9af0AE99174b3FAaB87174E31634D135f6d

New Contract Address :- 0x10adF50e15611d5A4DE3bd21F0DB7f3491A8ae0f

How Much Fee Paid For Contract Address Change Or Update Please Refer Recent PR #19387

* Update info.json

* Delete logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

ratscoin-dev committed:
Add Ratscoin (BEP20) (#19488)

* Add Ratscoin logo and info

Upload ratscoin's logo

Smartchain BEP20 token
[0x57b798d2252557f13A9148A075a72816f2707356]

* change info

* Update ratscoin.json

* Info for token Ratscoin

* Delete ratscoin.json

* Delete ratscoin.png

* Update info.json

* Update info.json

* Update logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Tdeveloper2022 committed:
Add Dabb Doge  (BEP20) (#19493)

* Add Dabb Doge(BEP20) info&logo

* Add Dabb Doge(BEP20) info&logo

DABB DOGE is a decentralized peer-to-peer digital currency that enables instant transactions to anyone anywhere in the world without any transaction fees. Dabb Doge has joined few NGOs to help the stray dogs either who had an accident or are left stranded by their owner. Team has been able to rescue 487 Dogs, which gave birth to 176 puppies.

* Delete info.json

* Delete logo.png.png

* Create logo.png

* Delete logo.png.png

* Update info.json

* Update info.json

* Update info.json

* Info for token Dabb Doge

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

cryptomanz1 committed:
Add new blockchain Cronos (CRC20) (#19497)

* Create info.json

* Create logo.png

* Update info.json

* Update info.json

cryptomanz1 committed:
Revert "Add new blockchain Cronos (CRC20) (#19497)" (#19512)

This reverts commit 970a1a3f1178a9b7c36b5b5781ed01962119be57.

Co-authored-by: Viacheslav Kulish <32914239+vcoolish@users.noreply.github.com>

dhabita committed:
MBCH (#19520)

* mbch

* Update info.json

* Update info.json

* Update info.json

* tes

* d

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

cryptomanz1 committed:
Delete ESDTSFT tokens (#19502)

* Delete logo.png

* Update info.json

* Delete logo.png

* Update info.json

* Delete logo.png

* Update info.json

* Delete logo.png

* Update info.json

cryptomanz1 committed:
Add Cronos (CRO) (#19528)

* Create info.json

* Create logo.png

* Update logo.png

* Update logo.png

appen111 committed:
Info for token Safe Energy (#19558)

Co-authored-by: hewigovens <360470+hewigovens@users.noreply.github.com>

anxolin committed:
Info for token COW (#19385)

Co-authored-by: Alex V <80408352+avsavsavs@users.noreply.github.com>

cryptomanz1 committed:
Change name for dapps 1inch (#19577)

* Delete 1inch.exchange.png

* Create app.1inch.io.png

TeddyDogeAdmin committed:
Add Teddy Doge (BEP20) (#19559)

* Add logo for trust wallet

check for png and josn  thx！

* Add trust wallet logo

Pls check jpg and josn，THX
Any problem touch telegram id:@TeddyDoge_Admin

* Add TEDDT logo on trust wallet

Add TEDDT logo on trust wallet
Telegram id：https://t.me/TeddyDoge_Admin

* Delete info.json

* Delete logo.png

* Add TEDDY logo on trust wallet #19547 

 Add TEDDY logo on trust wallet #19547

* Info for token Teddy Doge

* Delete info.json

* Delete logo.png

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

cryptomanz1 committed:
status abandoned for PathFundV1 (#19614)

* Update info.json

* Delete logo.png

cryptomanz1 committed:
Migration for PathFundV2 (BEP20) (#19615)

* Create logo.png

* Create info.json

huyneloworld committed:
Update info and logo of NELO (BEP20) (#19591)

Update latest information and logo of NELO token

Commit:
Fixes (sanity and consistency, auto)

Commit:
Roll social money updates February 2022 (#18258)

bjt54 committed:
Fixes (sanity and consistency, auto)

BobaBrewery committed:
Info for token Brewery (#19618)

0xllaCh committed:
Feature/add lfg v2 (#19540)

* Add LFG version 2 token info

* Update per requirements

* remove cross-chain tag

Commit:
Fixes (sanity and consistency, auto)

beneverrise committed:
Update EverRise to v3 (#19530)

* Update EverRise to v3

* Delete deprecated assets

* Revert "Delete deprecated assets"

This reverts commit 074cbe933166f8157f27ab81a03d5efcf31f41e9.

* Just remove deprecated icons

* 256x256 logos

Commit:
Fixes (sanity and consistency, auto)

bjt54 committed:
Add STEPN (BEP20) (#19637)

* Info for token STEPN

* Update info.json

Update token name

* Updated decimal value

Co-authored-by: Artem <725247+ArtemGoryunov@users.noreply.github.com>

Hectortheman committed:
Add MiniFootball (BEP20) (#19610)

* Info for token MiniFootball

* Update logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Satoshiisgod committed:
Add AirCash(aircash.finance) logo (#19648)

Add AirCash(aircash.finance) logo

(1) Dapp name: AirCash

(2) What does the Dapp do:
AirCash is the first and largest decentralized OTC platform in the galaxy.

(3) Dapp description:
AirCash aims to be the best solution for crypto cash in and out.

AirCash is the first and largest decentralized crypto-fiat OTC trading platform in the galaxy.

You can buy and sell crypto with fiat money in your decentralized wallet through AirCash.

(4) Dapp URL: http://aircash.finance

(5)Dapp Category: Exchange

(6) Certified:
I certify that this Dapp has been tested in Trust Wallet's browser for functional and visual defects.

(7) Telegram:
ANN - https://t.me/aircoinrealchannel,
Chat - https://t.me/aircoinofficialgroup,
DM Person - https://t.me/Aircoinrealassistant

(8) Email: hello@aircash.finance

Benson0224 committed:
Add APENFT token(BEP20 and ERC20) (#19566)

* Add APENFT token(BEP20 and ERC20)

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

ArtemGoryunov committed:
Fixes (sanity and consistency, auto)

Ukangit committed:
Add Ukan (UKAN) (ERC20) (#19652)

* Info for token Ukan (UKAN)

* Update logo.png

* Update info.json

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

cryptomanz1 committed:
Migration for Slam (BEP20) (#19675)

* Update info.json

* Delete logo.png

slamtoken committed:
Add Migrated contract of SLAM (#19673)

* Add Migrated contract of SLAM

SLAM is migrating to a new contract. Please remove or mark the old contract (0xcCe7F9eB881248E04f2975a3Fb3B62631ad9eE37)

Confirmation: https://slamtoken.com/migration

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

khoa1419 committed:
Add MYRA logo and info (#19687)

* Add MYRA logo and info

Add MYRA logo and info

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

WagmiToken committed:
Info for token WAGMI (#19692)

METAWHALEWORLD committed:
Change MTW Logo (#19698)

* Add Logo and Information

Please Add Logo and information

* Add Logo and Information

Add Logo and Information

* Add Logo and Information

Please Add Logo and Information

* Add Logo and Update Info

Please add MTW Logo and Update Info

Fee Paid References PR No - #19007

* Update info.json

* Add Logo and Update Info

Add MTW Logo and Update Info

Fee already paid PR No #19007

* Add  MTW Logo and Update Info

 Add MTW Logo and Update Info

MTW Is BEP20 Token

Fee Already Paid PR No #19007

* Add MTW Logo and Update Info

Add MTW Logo and Update Info

Fee Already Paid PR No #19007

* Add MTW Logo and Update Info

Add MTW Logo and Update Info

Fee Already Paid PR No #19007

All Changes are Done

MTW is BEP20 Token

Holder and Transaction are checked for References https://bscscan.com/token/0xD3Ac199E6e6A1668ED6566b6F6dcdf7641868731

* Update info.json

* Add MTW Logo and Update Info

Add MTW Logo and Update Info

Fee already Paid PR No #19007

Token Holders and Transactions are Checked Link :-https://bscscan.com/token/0xD3Ac199E6e6A1668ED6566b6F6dcdf7641868731

* Info for token Hare Plus

* Delete logo.png

* Add MTW Logo and Info

Add MTW Logo and Update Info

Fee Already Paid PR No #19007

All Changes are Done

MTW is BEP20 Token

PR #19053 

Required Smart Contract Audit Report Here is the link https://drive.google.com/file/d/1jTF5slWqhkMkE4yXoUm-ofLN0qvGpQdc/view?usp=sharing

Please Do as early as Possible and add logo on trust wallet

* Change MTW Logo

Please Change MTW Logo

Fee Already Paid We Need To Change Logo

Fee Paid Ref PR #19007

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: ramsty <91697270+ramsty@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

andreibancioiu committed:
Add top Elrond assets (plus some cleanup) (#19513)

* Add token definitions. Alter type, use METAESDT, where applicable.

* Add logo files.

* Fix after self-review.

* Cleanup (remove meta esdt).

* Add CRU, ZPAY.

* Add ecosystem pairs.

* Fix links.

byqm68 committed:
Add FileSystemVideo (BEP20) (#19709)

* Info for token FileSystemVideo

* Update logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

cryptomanz1 committed:
Add Ariva (ERC20) (#19765)

* Create info.json

* Create logo.png

samuel-tifi committed:
Add TiFi Token (BEP20) (#19743)

* Add TIFI logo and info

Hi, TWT Team,
Please kindly review and let me know for any updates.
Thanks!

* Update info.json

* Info for token TiFi Token

castellocoin committed:
Info for token Castello Coin (CAST) (#19764)

BOTDeFi committed:
Add BOT (BEP20) (#19786)

* Info for token BOT

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Update info.json

Cryptocool1 committed:
Delete logo.png

moonshot-platform committed:
Add Moonshot (BEP20) (#19734)

* Add Moonshot (v2) BEP20

* Info for token Moonshot

* Update info.json

Co-authored-by: Moonshot <>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
change contract for Moonshot (BEP20) (#19794)

* Update info.json

* Delete logo.png

StrongBlock committed:
Add Stronger (STRNGR) (ERC20) (#19745)

* Info for token Stronger (STRNGR)

* Update info.json

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

happytreasure-jump committed:
Add JGB (BEP20) (#19766)

* Info for token JGB

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

nodesbygirls-validator committed:
Osmosis: Validators: Add NodesByGirls (#19811)

w0dm4n committed:
Kryxivia (#19825)

Chester-Ngo committed:
Add logo and info Metamate (#19839)

* Add logo and info Metamate

Add logo and info Metamate

* Update logo

rosasalberto committed:
Add Gamium (BEP20) (#19814)

* Info for token Gamium

* Update info.json

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

tradeStrikeX committed:
Add StrikeX (BEP20) (#19871)

* StrikeX logo and info

* StrikeX logo and info upload

* duplicate

* StrikeX info and logo

* Delete info.json

* Update info.json

* Delete logo.png

* Update info.json

* Update info.json

* Update info.json

* update info.json

* Info for token StrikeX

* Delete logo.png

* Delete info.json

* upload logo.png

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>
Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

sunflowerstake committed:
Add Sunflower cosmos validator (#19888)

Co-authored-by: sunflowerstake <admin@sunflowerstake.com>

agverous committed:
feat: add anon asset info (#19891)

* feat: add anon asset info

* feat: update anon asset info

* feat: update tags

Co-authored-by: anon <anon>

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

WagmiToken committed:
Add WAGMI Game (WAGMI) (ERC20) (#19897)

* Info for token WAGMI Game (WAGMI)

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

naviocean committed:
Rebranding & Update new logo BHO (BEP20) (#19864)

* Add BHO (BEP20)

* Add BHO (BEP20)

* Update logo size to 256x256

* Rebranding & update new logo

* Add tags

* Update tags

Cryptocool1 committed:
Fixes (sanity and consistency, auto)

bjt54 committed:
update TT logo (#19910)

bjt54 committed:
Update TT name (#19911)

lenohard committed:
Info for token MetaSwapToken (#19894)

cryptomanz1 committed:
Change status for Elrond  (#19917)

* Create info.json

* Create info.json

* Create info.json

* Create info.json

AyushBherwani1998 committed:
Add Boba logo and info (#19896)

* Add Boba logo and info

* Updates boba logo size

naviocean committed:
Update BHO logo (BEP20) (#19921)

* Add BHO (BEP20)

* Add BHO (BEP20)

* Update logo size to 256x256

* Rebranding & update new logo

* Add tags

* Update tags

* Update logo.png

tw-covain committed:
Remove algorand from staking (#19936)

divinergames3 committed:
Info for token DIVINER (#19900)

GOMAFinance committed:
Add GOMA (BEP20) (#19812)

* Adding the TORII FINANCE LOGO

* Info for token GOMA

* Delete info.json

* Delete logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

cryptomanz1 committed:
Migration for GOMA (BEP20) (#19940)

* Update info.json

* Delete logo.png

Aggletoken committed:
Info for token aggle.io (#19938)

web3doge committed:
Add WEB3 Game (BEP20) (#19957)

* Info for token WEB3 Game

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

cryptomanz1 committed:
Migration for GHC (BEP20) (#19958)

* Update info.json

* Delete logo.png

sumner770 committed:
Request to add GHC logo (#19954)

Commit:
Fixes (sanity and consistency, auto)

Cryptocool1 committed:
Merge branch 'master' of https://github.com/trustwallet/assets

Sanji-Inu committed:
Add Sanji Inu (BEP20) (#19937)

* Info for token Sanji Inu

* Update logo.png

* Update logo.png

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>
Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Cryptocool1 committed:
Info for token Wrapped TON Coin (#19968)

AyushBherwani1998 committed:
Add metis chain info and logo (#19959)

hewigovens committed:
[Internal] Add GST token and update GMT info (#19990)

* Add GST token and update GMT info

* add tags

* update block explorer url

lordtoken-ltt committed:
Info for token LordToken (#19984)

* Info for token LordToken

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

ChainEngineer1 committed:
Add XCN in BSC (#19906)

vancattoken committed:
Migration for VANCAT Token (BEP20) (#19996)

* Migration for VANCAT Token (BEP20)

* Edit Coingecko link for VANCAT

* Delete logo.png

* Create logo.png

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

cryptomanz1 committed:
Delete logo.png (#20024)

NamTran9999 committed:
Add VicMove (BEP20) (#20021)

* Info for token VicMove

* Update info.json

* Update info.json

* Update logo.png

* Update info.json

cryptomanz1 committed:
Update logo.png (#20051)

DevEJR committed:
Add ElonBank Logo and information (#19993)

* Add ElonBank Logo and information

Hello,

We would like to add our logo. Our token is 1 month old with an audited contract.

Thanks,

* Update Json

Some lines fixed

* Update info.json

* Update info.json

Co-authored-by: Nazar <85495047+cryptomanz@users.noreply.github.com>

Commit:
Fixes (sanity and consistency, auto)

zoroinu committed:
Add Zoro Inu (BEP20) (#20054)

* Add ZORO logo and info

Dear team, we want to add logo and info to trust wallet, please help us.

* Add ZORO logo and info

Dear team, we want to add logo and info to trust wallet, please help us.

* Add ZORO logo and info

Dear team, we want to add logo and info to trust wallet, please help us.

* Add ZORO logo and info

Dear team, we want to add logo and info to trust wallet, please help us.

+ Whitepaper: https://docs.zoroinu.io/
+ Telegram chat: https://t.me/Zoroinucommunity
+ Telegram channel: https://t.me/Zoroinuworld
+ Twitter: https://twitter.com/ZoroToken
+ Medium: https://zoroinu.medium.com/
+ Github: https://github.com/zoroinu
+ Coinmarketcap: https://coinmarketcap.com/currencies/zoro-inu/
+ Coingecko: https://www.coingecko.com/en/coins/zoro-inu

Best regards !

* Info for token Zoro Inu

* Update info.json

Co-authored-by: CryptoA <82717856+Cryptocool1@users.noreply.github.com>

AyushBherwani1998 committed:
Adds aurora info and logo (#20068)

OnePanthera committed:
Add files via upload (#20073)

Commit:
Fixes (sanity and consistency, auto)

cryptomanz1 committed:
Migration for Catcoin V2 (#20096)

* Update info.json

* Delete logo.png

Iamdeadlyz committed:
Add Decentraland MANA BEP20 (#20110)

Adding the BEP20 version of Decentraland (MANA)

Token was created by Binance's BEP20 factory contract: https://bscscan.com/tx/0xdab292b7ff48b270714284489870467094995161b72fbd852dc5d05d09151c85

PankajTMB committed:
Info for token KISSAN (#19983)

chk0912 committed:
Merge branch 'master' of github.com:trustwallet/assets into merge/from_main_repo

# Conflicts:
#	blockchains/ethereum/tokenlist.json

Adding:
- Avalance assets
- Other networks that we are not using right now
- Mainnet assets
- Fantom assets
- Polygon assets

Updates:
- BSC -> BNB
- Changes for mainnet tokens and logos

ElectricJeans reviewed (commented):
lgtm

ElectricJeans reviewed (approved):
lgtm
